### ISSUE TITLE
CRTF region reading failure

### ISSUE THREAD

**Issue by keflavich** [astropy/regions#405]
This is another CARTA-written CRTF file.  We need to support the `labelcolor` keyword.

File `test.crtf`:
```
#CRTFv0 CASA Region Text Format version 0
poly [[238.29839664deg, -54.60647979deg], [238.29935543deg, -54.60610099deg], [238.30011816deg, -54.60598728deg], [238.30035808deg, -54.60646691deg], [238.29933401deg, -54.60708556deg], [238.29868031deg, -54.60743908deg], [238.29820094deg, -54.60771684deg], [238.29728571deg, -54.60814611deg], [238.29626149deg, -54.60858802deg], [238.29523727deg, -54.60908040deg], [238.29423473deg, -54.60920672deg], [238.29399489deg, -54.60885331deg], [238.29440888deg, -54.60841148deg], [238.29510618deg, -54.60793175deg], [238.29560731deg, -54.60741417deg], [238.29589059deg, -54.60726267deg], [238.29669691deg, -54.60707325deg], [238.29721997deg, -54.60704794deg], [238.29783007deg, -54.60665657deg]] coord=ICRS, corr=[I], linewidth=2, linestyle=-, symsize=1, symthick=1, color=2ee6d6, font=Helvetica, fontsize=10, fontstyle=bold, usetex=false, label="region18", labelcolor=green, labelpos=top
```

```python
regions.Regions.read('test.crtf')
Traceback (most recent call last):
  File "<ipython-input-15-bb42a7655814>", line 1, in <module>
    regions.Regions.read('test.crtf')
  File "/orange/adamginsburg/miniconda3/envs/python39/lib/python3.9/site-packages/regions/core/regions.py", line 160, in read
    return RegionsRegistry.read(filename, cls, format=format,
  File "/orange/adamginsburg/miniconda3/envs/python39/lib/python3.9/site-packages/regions/core/registry.py", line 76, in read
    raise IORegistryError(msg) from None
IORegistryError: No reader defined for format "crtf" and class "Regions".

The available formats for the Regions class are:

Format Parse Serialize Read Write Auto-identify
------ ----- --------- ---- ----- -------------
  crtf   Yes       Yes  Yes   Yes           Yes
   ds9   Yes       Yes  Yes   Yes           Yes
  fits   Yes       Yes  Yes   Yes           Yes
```

```python
regions.io.crtf.read.read_crtf('test.crtf')
WARNING: AstropyDeprecationWarning: The read_crtf function is deprecated and may be removed in a future version.
        Use `regions.Regions.read` instead. [warnings]
Traceback (most recent call last):
  File "<ipython-input-17-892939c6132c>", line 1, in <module>
    regions.io.crtf.read.read_crtf('test.crtf')
  File "/home/adamginsburg/.local/lib/python3.9/site-packages/astropy/utils/decorators.py", line 125, in deprecated_func
    return func(*args, **kwargs)
  File "/orange/adamginsburg/miniconda3/envs/python39/lib/python3.9/site-packages/regions/io/crtf/read.py", line 549, in read_crtf
    return _read_crtf(filename, errors=errors, cache=cache)
  File "/orange/adamginsburg/miniconda3/envs/python39/lib/python3.9/site-packages/regions/io/crtf/read.py", line 80, in _read_crtf
    return _parse_crtf(region_string, errors=errors)
  File "/orange/adamginsburg/miniconda3/envs/python39/lib/python3.9/site-packages/regions/io/crtf/read.py", line 89, in _parse_crtf
    return Regions(parser.shapes.to_regions())
  File "/orange/adamginsburg/miniconda3/envs/python39/lib/python3.9/site-packages/regions/io/core.py", line 122, in to_regions
    region = shape.to_region()
  File "/orange/adamginsburg/miniconda3/envs/python39/lib/python3.9/site-packages/regions/io/core.py", line 694, in to_region
    reg.meta[key] = self.meta[key]
  File "/orange/adamginsburg/miniconda3/envs/python39/lib/python3.9/site-packages/regions/core/metadata.py", line 37, in __setitem__
    raise KeyError(f'{key} is not a valid key for this class.')
KeyError: 'labelcolor is not a valid key for this class.'
```

**Comment by keflavich** [comment]
uh, hm.  We support `labelcolor`.  I don't know why the issue is being raised.

**Comment by apmtowner** [comment]
Was any progress ever made on this issue? I just ran into it again. As before, I can solve it be manually deleting all the labels from my regions in the .crtf files, but it would be nice to not *have* to.

**Comment by larrybradley** [comment]
Fixed in #473.
